Helper_Scripts/TTS_Installers/install_tts_pocket_tts_onnx.py

```
@@ -0,0 +1,465 @@
+    Returns:
+        Resolved config path if found, otherwise None.
+    """
+    if config_path_arg:
+        return Path(config_path_arg).expanduser()
+    repo_root = _resolve_repo_root()
+    if repo_root and str(repo_root) not in sys.path:
+        sys.path.insert(0, str(repo_root))
+    try:
+        from tldw_Server_API.app.core import config_paths
+        return config_paths.resolve_module_yaml("tts")
+    except Exception:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)